all2vec/__init__.py

```diff
@@ -217,21 +217,22 @@ def save(self, folder):
             json.dump(enttypes, handle)
 
     @classmethod
-    def load(cls, folder):
-        """Load object."""
+    def load(cls, file_getter):
```

Comment:
This is a breaking API change for existing code that calls `.load` with a file path. You can avoid this by taking advantage of duck typing:

```
def load(cls, file_getter_or_folder):
    try:
        with file_getter_or_folder.get_file('entity_info.json') as f:
            # open file
    except AttributeError:
        with open(file_getter_or_folder) as f:
            # open file
```